src/Uno.UWP/ApplicationModel/Chat/ChatMessageManager.iOS.cs

```diff
@@ -0,0 +1,43 @@
+﻿#if __IOS__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MessageUI;
+using UIKit;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Chat
+{
+	public partial class ChatMessageManager
+	{
+		public static IAsyncAction ShowComposeSmsMessageAsync(ChatMessage message)
+		{
+			if (message == null)
+			{
+				throw new ArgumentNullException(nameof(message));
+			}
+
+			var window = UIApplication.SharedApplication.KeyWindow;
+			var controller = window.RootViewController;
+
+			var messageController = new MFMessageComposeViewController();
+
+			messageController.Body = message.Body;
+			messageController.Recipients = message?.Recipients?.ToArray() ?? new string[] { };
+
+			var tcs = new TaskCompletionSource<bool>();
+			messageController.Finished += (sender, e) =>
```

Comment:
What happens if the user cancels sending the message? What if there's an error? (No network, invalid recipient, etc.) The task should still complete.

Reply:
The truth is I have closely followed the implementation of SMS in Xamarin.Essentials and this was the approach chosen there. I will check with UWP and try to do the same thing